app/components/chat_form/chat_form.css

```
@@ -4,6 +4,19 @@
   align-items: flex-end;
 }
 
+.ChatForm h3 {
+  width: 100%;
```

```suggestion
  align-self: flex-start;
```
Is there any other reason that you want this to be 100% width other than aligning it to the left? It looks to me that the **ChatForm** is a flex element that is right-aligned with `align-items: flex-end;` We can align any of the child elements different from the other with `align-self`